chia/daemon/server.py

```diff
@@ -1087,7 +1087,7 @@ async def start_service(self, request: Dict[str, Any]):
             else:
                 self.log.info(f"Service {service_command} already running")
                 already_running = True
-        elif service_command in self.connections:
```

Comment:
is there a good reason we sometimes don't have this member? is there an important distinction between having the member and having it as an empty list?

Reply:
I think that we 'always' have `self.connections` as an attribute, are you suggesting that the `self.connections` `dict` could be initialized with all the services and empty lists?  It looks like not-in-the-`dict` vs. empty list is just whether the service has ever been connected to this specific daemon process.

It may be that this new block should have ended up living in `.is_running()` instead of here.

Reply:
oh, I think I misread this. `self.connections` is a Dict mapping "commands" to a list. the current logic doesn't work because we still have the dict entry when the list is empty.